elide-contrib/elide-dynamic-config-helpers/src/test/java/com/yahoo/elide/contrib/dynamicconfighelpers/parser/ElideConfigParserTest.java

```diff
@@ -41,13 +42,15 @@ public void testValidateVariablePath() throws Exception {
             assertEquals(t.getMeasures().get(0).getName() , t.getMeasures().get(0).getDescription());
             assertEquals("MAX(score)", t.getMeasures().get(0).getDefinition());
             assertEquals(Table.Cardinality.LARGE, t.getCardinality());
+            // test hydration, variable substitution
+            assertEquals(Type.INTEGER, t.getMeasures().get(0).getType());
         }
     }
 
     @Test
     public void testNullConfig() {
         try {
-            testClass.parseConfigPath(null);
+//            testClass.parseConfigPath(null);
```

Comment:
Is this intended?

Reply:
resolved in https://github.com/yahoo/elide/pull/1210